### PR TITLE
Max width updates, drop news sidebar down

### PR DESCRIPTION
**Pull request by JMStewart00**
## Summary / Approach

Lots to sift through and clean up but for the most part just created some overrides using parent selectors. 
Separated the article stuff into it's own file, even if it's a little late to start doing that. 
Messed around with some breakpoints stuff and cleaned up the node page template.

## Testing Instruction(s)

Use the Multidev to visit pages for all the content types. 
- News Articles
- Organization Pages
- Board/Commission
- Service Guides

## Screenshots

No need now with Multidev.

## Metadata

| Question | Answer |
|----------|--------|
| Did you perform a self-review of this PR? | yes
| Documentation reflects changes? | n/a
| Unit/Functional tests reflect changes? | n/a
| Did you perform browser testing? | yes, FF and Chrome
| Did you provide detail in the summary on how and where to test this branch? | yep
| Relevant links | JIRA issue, bug reports, security alerts, etc.
| LG-105 | https://apaxsoftware.atlassian.net/browse/LG-105
| LG-120 | https://apaxsoftware.atlassian.net/browse/LG-120

### CODE REVIEW

**Comment by TaylorAkin** [comment]
[Visit Site](https://pr-413-lexky-d8.pantheonsite.io/)

Created multidev environment [pr-413](https://dashboard.pantheon.io/sites/085efc9d-6f4f-4d02-a80b-9d5cf086a982#pr-413) for lexky-d8.

**Comment by TaylorAkin** [comment]
Screenshot 2023-07-07 at 9 30 00 PM
The 1st news card barley overflows out of the fixed width for the container, so it looks like it doesn't have left border. This is on a 1800px width screen.

**Comment by TaylorAkin** [comment]
Screenshot 2023-07-07 at 9 34 06 PM
Screenshot 2023-07-07 at 9 34 06 PM (2)
This is the front page

**Comment by JMStewart00** [comment]
Alright. Stuff like that is going to happen as I have no idea where these blocks show up. Appreciate you catching them. I didn't see the overflow in my testing but I'll take care of it quickly tomorrow.

**Comment by TaylorAkin** [comment]
Screenshot 2023-07-07 at 9 39 37 PM
Also making sure the "More News" link button stays centered

File: web/themes/custom/lex/scss/molecules/_news_card.scss [hunk 1]
```diff
@@ -173,7 +178,8 @@
 }
 
 .news-container,
-#block-views-block-front-page-news-block-2 {
+#block-views-block-front-page-news-block-2,
+#block-views-block-front-page-news-block-2-2 {
```
Comment (TaylorAkin) [inline web/themes/custom/lex/scss/molecules/_news_card.scss:182]: added this guy

File: web/themes/custom/lex/templates/pages/page.html.twig [hunk 2]
```diff
@@ -14,24 +14,24 @@
 #}
 
 {% block page_alerts %}
-  {{ page.alerts }}
+	{{ page.alerts }}
```
Comment (TaylorAkin) [inline web/themes/custom/lex/templates/pages/page.html.twig:17]: Idk why 2 space indents are looking like 6 here. 🤷

**Review by TaylorAkin — APPROVED** [review]

File: web/themes/custom/lex/scss/molecules/_news_card.scss (hunk 1 above)
Comment (JMStewart00) [reply web/themes/custom/lex/scss/molecules/_news_card.scss:182]: Don’t love it but it is what it is.

File: web/themes/custom/lex/templates/pages/page.html.twig (hunk 2 above)
Comment (JMStewart00) [reply web/themes/custom/lex/templates/pages/page.html.twig:17]: Yeah I was having that issue too. Weird.

**Pull request merged**